Automattic/jetpack-onboarding
# Business Address Step

Prefill `business_name` with `site_title`

Fixes #33 

To test:
- checkout this branch, and reset your jpo options
- go through the business flow
- ensure that the business name is pre-filled with your site title

<img width="1142" alt="screen shot 2016-08-30 at 10 12 17 am" src="https://cloud.githubusercontent.com/assets/2694219/18092442/5047e942-6e9a-11e6-9a95-6c70f568a5cb.png">

cc: @jessefriedman
